packages/charting/src/components/VerticalBarChart/VerticalBarChart.types.ts

```diff
@@ -4,15 +4,14 @@ import {
   ICartesianChartProps,
   ICartesianChartStyleProps,
   ICartesianChartStyles,
-  IDataPoint,
   IVerticalBarChartDataPoint,
 } from '../../index';
 
 export interface IVerticalBarChartProps extends ICartesianChartProps {
   /**
    * Data to render in the chart.
    */
-  data?: IDataPoint[];
+  data?: IVerticalBarChartDataPoint[];
```

Comment:
Please change the version to major as this is changing the Interface. Else, to keep consistent change this to IDataPoint and copy this to IVerticalBarChartDataPoint[]

Reply:
actually both have same props. look at the snaps below
![image](https://user-images.githubusercontent.com/33802398/96618425-8d937880-1322-11eb-99f0-5839694150ee.png)


![image](https://user-images.githubusercontent.com/33802398/96618446-97b57700-1322-11eb-8501-c44ffea05a3d.png)


anything which is not common in both of them, those are optional.. that is why there is no need to change it to major